lms/routes.py

```diff
@@ -250,6 +250,11 @@ def includeme(config):  # noqa: PLR0915
         factory="lms.resources.dashboard.DashboardResource",
     )
 
+    config.add_route(
+        "api.dashboard.organizations.courses",
+        "/api/dashboard/organizations/{public_id}/courses",
+    )
```

Comment:
I guess we also need a non-API route to be registered, that returns the "blank" template for the FE app to kick-in

Reply:
Yup :+1: